tests/index.test.js

```diff
@@ -1,16 +1,93 @@
 require('../config.js')
-const test = require('tape')
+const tape = require('tape')
 const dbConnection = require('../db/connect.js')
-// const server = require('../src/server.js')
-// const supertest = require('supertest')
-// const Users = require('../db/models/User.js')
-// ^ will use these in the futue
+const server = require('../src/server.js')
+const supertest = require('supertest')
+const User = require('../db/models/User.js')
+const testData = require('./fixtures/users.json')
+
+// function to drop collection and end the tests
+const dropCollectionAndEnd = (myCollection, t) => {
+  myCollection.collection.drop()
+    .then(() => {
+      t.end()
+    })
+    .catch(err => t.end(err))
+}
 
 dbConnection.once('open', () => {
-  require('./test_1.test.js')
-  require('./test_2.test.js')
+  tape('test /users when nothing in database', (t) => {
```

Comment:
Yeah, you need to move this into its own file.

Prob: `test/controllers/user.test.js`